.eslintrc.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   plugins: ['@typescript-eslint'],
   rules: {
+    'no-undef': 0,
```

Comment:
Why to turn off this?

Reply:
I couldn't get the error to go away, but after adding `plugin:@typescript-eslint/recommended` it's fixed now